### DISCUSSION TITLE
"pintura" feita com puro HTML e CSS

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#1061] [Dicas]
OK, depois de ver isso fico me perguntando agora se eu realmente sei CSS, agora existe uma crise existencial dentro de mim:
-  será que estou na profissão certa?
-  será que existe pão no céu?

http://diana-adrianne.com/purecss-zigario/

**Comment by renanlara** [comment]
Poxa, super legal! Eu penso a mesma coisa também, mas pra ser bem sincero, nem tudo na vida gira em torno de uma coisa só. Existem diversas aplicações para tudo na vida. Vai do interesse de cada um onde irá querer seguir a sua trilha.

Se eu fosse listar aqui tudo o que eu gostaria de aprender ou melhor ainda, **precisaria** aprender... Acredito que não deveria estar aqui fazendo parte da comunidade. Mas mesmo assim, sustento minha família com o meu trabalho.

**Não desanima cara! Abraço.**

**Comment by leandroruel** [comment]
hauhauah valew é so uma brinks mesmo com a imagem, que aliás é super FODA! depois vou ate parabenizar a pessoa

**Comment by kvnol** [comment]
Vi isso pelo Twitter e: realmente SENSACIONAL.

**Comment by matheusml** [comment]
É amigos. Não sei CSS.

**Comment by simaodeveloper** [comment]
Maravilhoso, mas não foi feito em um dia 💃

**Comment by didiraja** [comment]
Inacreditável. Como sempre digo: **CSS é magia negra.**

**Comment by igorthurow** [comment]
Incrível

**Comment by giovannism20** [comment]
https://github.com/cyanharlow/purecss-zigario
